Makefile

```diff
@@ -10,3 +11,5 @@ test:
 	make -C src/api test
 	make -C dist test
 	make -C src/backend test
+clean: 
+	rm -rf ./docs/api/html
```

Comment:
I really like the idea of having a "make clean", but I would prefer `rm -rf ....` in src/api/Makefile and just call 
```
make -C src/api clean
```
  to stay consistent with the rest of the Makefile and give developers more choice, what component to clean.